docs/get-started/features.md

```diff
@@ -61,7 +61,11 @@ The **cpu** feature source supports the following labels:
 | cpuid                   | &lt;cpuid flag&gt; | CPU capability is supported
 | hardware_multithreading |                    | Hardware multithreading, such as Intel HTT, enabled (number of logical CPUs is greater than physical CPUs)
 | power                   | sst_bf.enabled     | Intel SST-BF ([Intel Speed Select Technology][intel-sst] - Base frequency) enabled
-| [pstate][intel-pstate]  | turbo              | Set to 'true' if turbo frequencies are enabled in Intel pstate driver, set to 'false' if they have been disabled.
+| [pstate][intel-pstate]  | enabled            | Set to 'true' if the Intel pstate driver is in use, otherwise set to 'false'.
+|                         | turbo              | Set to 'true' if turbo frequencies are enabled in Intel pstate driver, set to 'false' if they have been disabled.
+|                         | powersave          | Set to 'true' if the scaling policy is set to 'powersave'.
+|                         | performance        | Set to 'true' if the scaling policy is set to 'performance'.
```

Comment:
Again, just thinking should we only have one `governor` label that would be set to the active scaling_governor, assuming all cpus (policies) use the same governor. This label would not exist if different policies used different governors. I doubt that there is any practical use case in K8s scope to have the system partitioned by the scaling governor (at least scheduling on the desired cpus would be very hard)?

Another consideration: the practical operation of the scaling governor depends quite heavily on the mode (active/passive). Should these labels only exist if in the "active" mode?

WDYT?

Reply:
Sure, will change to have one label instead of two.  Maybe call it "policy" instead of "governor"?  It will be set to the value of scaling_governor. And yes also it should only be set if the status of intel_pstate is active according to https://www.kernel.org/doc/html/v4.12/admin-guide/pm/intel_pstate.html.

Thank you for the review.